acrobat/blocks/eventwrapper/eventwrapper.js

```diff
@@ -86,6 +92,38 @@ export default function init(element) {
       }
     }
 
+    if (verb === 'rotate-pages') {
```

Comment:
ok to do in a separate PR, but don't we need the same rules for the new fill & sign?  we'd also have to be careful to coordinate both rollouts

Reply:
@peller when the widget for 'fill & sign' is ready, it will be easy to update the condition for 'rotate-pages' with `if (verb === 'rotate-pages' || verb === 'fillsign')` wherever applicable. That behaviour will be updated with this ticket https://jira.corp.adobe.com/browse/MWPW-133658.

Furthermore, in the last commit, I fixed the bug with styles in the case of a failing upload or when a corrupt file is uploaded.